.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,15 +6,24 @@ labels: 'needs: discussion'
 assignees: ''
 
 ---
+<!-- Thank you for contributing. These HTML commments will not render in the issue, but you can delete them once you've read them if you prefer! -->
 
-Those who maintain this project do so as volunteers and we have no control over their schedule or priorities.
+### Proposed change
+<!-- Use this section to describe the feature you'd like to be added. -->
 
-This means if the feature you are proposing is something you'd like to see added the best way to achieve that is for you to organise the effort required to build it. This could be evangelising for the feature, paying someone, doing it yourself, etc.
 
-Even so it might never get merged because the trade off between additional happy users and maintenance burden is not favourable.
+### Alternative options
+<!-- Use this section to describe alternative options and why you've decided on the proposed feature above. -->
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+### Who would use this feature?
+<!-- Describe the audience for this feature. This information will affect who chooses to work on the feature with you. -->
+
+
+### How much effort will adding it take?
```

Comment:
One idea here (feel free to ignore !), would be to ask folks to say _where_ they think that effort will be targeted, rather than just how much. It could help to avoid answers like "3 hours" and would make sense if contributors are more or less comfortable with different parts of the code base.

Reply:
I like this. Though also see the earlier discussion where we concluded that it almost doesn't matter what you write here. The goal is to get people thinking about the fact that there is no free lunch and not to get a realistic estimate (which is super hard even for practiced teams)

Reply:
(see below for a suggested addition)